### PR TITLE
Restructure Summary and rename to Overview.

### PR DESCRIPTION
**Pull request by rosiel**
**GitHub Issue**: none / attempt to improve docs.

# What does this Pull Request do?

Rewrites the first (landing) page of the islandora documentation. 

In some cases, I found the wording of the 'sales-pitch' part to be incorrect, outdated, or misleading. The new wording reflects my understanding of these components.

# What's new?

* Renames the first page from "Summary" to "Overview".
* Rewrites the sales-pitch part of the summary to use "grounding" keywords. Phrases it as though this collection of ways-of-doing-things (along with the community) "are" Islandora, while making things that are "optional" sound optional.
* Adds a link to the demo (should we?) and quickstart section to the summary page.

# How should this be tested?

* Are the statements about what Islandora is and does accurate, and sound good?
* can you use the demo and quickstart instructions? (are they useful?)

# Additional Notes:

* Is there a link to docker up the ISLE version yet? :D 

# Interested parties
@Islandora/8-x-committers

### CODE REVIEW

File: docs/index.md
Comment (manez) [inline docs/index.md]: I'd link to https://github.com/Islandora-Labs/Islandora-Cookbook instead of Islandora Awesome

**Comment by rosiel** [comment]
ping @Islandora/8-x-committers

**Review by manez — APPROVED** [review]
Looks good!

File: docs/index.md
Comment (mjordan) [inline docs/index.md]: I'm not comfortable saying Islandora is a "fully-functional preservation repository for digital assets". It can't replace Archivematica for most users of that system, for example. I think it's more accurate to call Islandora a Digital Assets Management System that has a lot of preservation capabilities.

Comment (mjordan) [reply docs/index.md]: Mind you, the only major thing Islandora can't currently do that Archivematica can is [enact configurable preservation policies](https://www.archivematica.org/en/docs/archivematica-1.11/user-manual/preservation/preservation-planning/#pres-policies). It could do it if we wanted it to.

File: docs/index.md
Comment (mjordan) [inline docs/index.md:10]: Controlled Access Terms is awesome but it's not a preservervation tool.

File: docs/index.md
Comment (rosiel) [reply docs/index.md]: [edit] Thanks so much, I'll reword that! (sorry for the defensive first draft of this comment, I misinterpreted where your comments were)

Comment (mjordan) [reply docs/index.md]: I'm happy to propose wording - should I do that here, or open another PR once this is merged?

File: docs/index.md
Comment (rosiel) [reply docs/index.md:10]: Agreed! That's why I took that line out.

Comment (mjordan) [reply docs/index.md:10]: Doh! Sorry, I didn't notice the `-`. Please mark as resolved.

File: docs/index.md
Comment (rosiel) [reply docs/index.md]: here's fine! I was thinking less "preservation repository" more "digital asset management system" but the rest of the sentence would need balancing out... what are you thinking?

Comment (mjordan) [reply docs/index.md]: Maybe

> [Islandora](https://islandora.ca) is an open-source framework that provides the necessary tools turn a [Drupal](https://www.drupal.org) website into a fully-functional Digital Assets Management System. 

And then add under the "Islandora:" section a note about its digital preservation capabilities. I've been informally maintaining https://github.com/mjordan/digital_preservation_using_islandora just to keep some of this stuff in one place, but so far, only Fedora and FITS have been mainstreamed into core Islandora. Riprap, Bagger, PREMIS and Reports have not yet undergone LSAP or adoption in to core. So not sure what level of detail to provide in the "Islandora:" section entry. Would be happy to hear what others think.

Comment (rosiel) [reply docs/index.md]: Seems like something that should be explained in the appropriate section of the documentation (and the non-core ones maybe getting more attention with this recent survey)

Comment (rosiel) [reply docs/index.md]: @mjordan that documentation about preservation is awesome, can I propose it be added to the [Cookbook](https://github.com/Islandora-Labs/Islandora-Cookbook)?

Comment (mjordan) [reply docs/index.md]: Sure, that would be great!

Comment (manez) [reply docs/index.md]: I'll do a PR for that

**Comment by rosiel** [comment]
ping?

**Review by dannylamb — APPROVED** [review]
Merging based on @manez's approval

**Comment by whikloj** [comment]
This PR seems to have lots of "notes" for possible changes. Those will now be public. Was that the plan or did we want those resolved first?

**Comment by whikloj** [comment]
Or does the `[//]` cause it to be hidden?
ie.
```
[//]: # (We should probably replace as many of the links in this section as possible with links to within this documentation, and make it clear which are internal links and which link out.)
```

**Comment by manez** [comment]
Hidden away in the built docs

**Pull request merged**
